lib/src/components/Radio/Radio.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { ForwardRefRenderFunction } from "react";
+import { IRadioProps } from "./radio.types";
+import {
+  radioWrapperClass,
+  radioCheckMarkClass,
+  radioThemeVars,
+  radioSizes,
+  radioThemeClass
+} from "./radio.styles.css";
+import "./radio.global.styles.css";
+import { assignInlineVars } from "@vanilla-extract/dynamic";
+import { eliminateUndefinedKeys } from "src/utils/object-utils";
+
+const SvgDot = () => {
+  return (
+    <svg viewBox="0 0 6 6">
+      <circle fill="currentColor" cx="3" cy="3" r="3"></circle>
+    </svg>
+  );
+};
```

Comment:
make a seperate component in `_internal`

Reply:
Done